UnityProject/Assets/Scripts/Objects/Disposals/DisposalBin.cs

```diff
@@ -35,28 +35,33 @@ public enum BinOverlaySprite
 	[ExecuteInEditMode]
 	public class DisposalBin : DisposalMachine, IServerDespawn, IExaminable, ICheckedInteractable<MouseDrop>
 	{
-		const int CHARGED_PRESSURE = 600; // kPa
-		const int AUTO_FLUSH_DELAY = 2;
-		const float ANIMATION_TIME = 1.3f; // As per sprite sheet JSON file.
+		private readonly int CHARGED_PRESSURE = 600; // kPa
```

Comment:
why can't these be configurable on inspector?

Reply:
Hmm, I tend to use constants when the value, once found, shouldn't change unless the code changes too. Though, some of those probably could be serialized.

Reply:
in that case, make it ``const`` or was codacy complaining? It should only complain about publics const that are used in other classes.

Reply:
You're right. Will fix.